src/pages/view/fb.js

```diff
@@ -0,0 +1,26 @@
+import { initializeApp } from "firebase/app";
```

Comment:
Please remove this file completely and make use of .env.local and the firebase.js file in the ./src scope